### PR TITLE
Governing body detail page

### PR DESCRIPTION
**Pull request by anitacaron**
It requires a change in the backend that is in this [PR](https://github.com/lblod/app-contact-hub/pull/20).

### CODE REVIEW

**Comment by anitacaron** [comment]
I'll update this to use the new card component!

**Review by Windvis — CHANGES_REQUESTED** [review]

File: app/templates/administrative-units/administrative-unit/governing-bodies/governing-body/index.hbs [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+                  @route="people.person"
+                  @model={{mandatory.governingAlias.id}}
+                >
+                  {{mandatory.governingAlias.familyName}}
+                </AuLink>
+              </td>
+              <td>
+                <AuLink
+                  @route="people.person.positions.position"
+                  @model={{mandatory.id}}
```
[the remark this answers is not in the document]
Comment (anitacaron) [reply app/templates/administrative-units/administrative-unit/governing-bodies/governing-body/index.hbs:201]: We discussed it yesterday. 😬

[the remark this answers is not in the document]
Comment (Windvis) [reply app/templates/administrative-units/administrative-unit/governing-bodies/governing-body/index.hbs:201]: Hmmmmm did we :no_mouth:. I only remember the breadcrumb issue. Le me check

[the remark this answers is not in the document]
Comment (anitacaron) [reply app/templates/administrative-units/administrative-unit/governing-bodies/governing-body/index.hbs:201]: 😬 You said the error is related to the breadcrumb, but I was saying about this link 😬

File: app/routes/administrative-units/administrative-unit/governing-bodies/governing-body.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        include: 'mandates,mandates.role-board,mandates.held-by',
+      }
+    );
+
+    let mandates = await govBodyTimeSpec.mandates;
+
+    let mandateLid = mandates.filter((mandate) => {
+      return (
+        mandate.get('roleBoard.label') ===
+        'Bestuurslid van het bestuur van de eredienst'
```
[the remark this answers is not in the document]
Comment (anitacaron) [reply app/routes/administrative-units/administrative-unit/governing-bodies/governing-body.js:26]: Yes, I'll change it.

File: app/templates/administrative-units/administrative-unit/governing-bodies/governing-body.hbs [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{{page-title "Bestuursorganen"}}
```
[the remark this answers is not in the document]
Comment (anitacaron) [reply app/templates/administrative-units/administrative-unit/governing-bodies/governing-body.hbs:1]: No, I forgot to change it.

File: app/routes/administrative-units/administrative-unit/governing-bodies/governing-body.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+      {
+        reload: true,
+        include: 'mandates,mandates.role-board,mandates.held-by',
+      }
+    );
+
+    let mandates = await govBodyTimeSpec.mandates;
+
+    let mandateLid = mandates.filter((mandate) => {
+      return mandate.get('roleBoard.label') === MEMBER_ROLE_ID;
```
Comment (Windvis) [inline app/routes/administrative-units/administrative-unit/governing-bodies/governing-body.js:26]: Should be `roleBoard.id` now I think?

**Review by Windvis — CHANGES_REQUESTED** [review]

File: app/routes/administrative-units/administrative-unit/governing-bodies/governing-body.js (hunk 4 above)
Comment (anitacaron) [reply app/routes/administrative-units/administrative-unit/governing-bodies/governing-body.js:26]: ops

**Pull request merged**